indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -37,4 +37,65 @@
 
         setupListGenerator(filterConfig);
     };
+
+    global.setupAbstractPage = function setupAbstractPage() {
+        $('body').on('indico:confirmed', '.judge-button', function(evt) {
+            var $box = $('#abstract-decision-box');
+            var $this = $(this);
+            var $form = $box.find('form');
+
+            evt.preventDefault();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                data: getFormParams($form),
+                success: function(data) {
+                    if (data.page_html) {
+                        $('.management-page').replaceWith(data.page_html);
+                        $box = $('#abstract-decision-box');
+                    } else {
+                        $box.html(data.box_html);
+                    }
+                    initForms($box);
```

Comment:
We don't need to. Once the abstract is judged, there are no more forms on the page. At least for now.